### DISCUSSION TITLE
Alert - this works but should it

### DISCUSSION THREAD

**Discussion by rhscjohn-dev** [themesberg/flowbite-svelte#85] [Q&A]
Alert Component.

I was playing around and seeing if I could use my own color for an Alert component.

Code
```
<Alert id="alert-red" class="bg-slate-200" icon={XCircleSolid} rounded>
  <span slot="content">
    <span class="font-medium"> Info alert! </span>
    Change a few things up and try submitting again.
  </span>
</Alert>
```

Note the lack of prop color.  

As you can see the background color is gray.
![Screenshot 2022-06-01 134458](https://user-images.githubusercontent.com/60523782/171469646-c64165a9-2961-4e1d-89e8-343b0d39a483.png)

With the use of a color prop throughout you components, you are limiting the dev  to whatever colors you allow for that prop.  If i define color called 'primary',  I can.t say color="primary" because primary is not an assignable type.

**Comment by shinokada** [comment]
Can you update to v0.24.15 and use `customBgClass`, `customBorderAccentClass`, and `customTextColors ` to customize your color.
